Colors

fasonju opened the pull request.

aleks-chitalov commented:
Linter is complaining to me about the files. Also if you can capitalize lables that would be nice :3. Should we add a lint check to the ci?

fasonju commented:
> Linter is complaining to me about the files. Also if you can capitalize lables that would be nice :3. Should we add a lint check to the ci?

I addressed these issues, idk if setting up a linter for string literals is feasible.

aleks-chitalov reviewed (approved):
thank you for adressing them!
